File: easy/26-remove_duplicates_from_sorted_array.cc
#include <unordered_map>
#include <vector>
using std::vector;
using std::unordered_map;
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        unordered_map<int, int> map;
        for(int i=0; i<nums.size(); i++){
            if(map.count(nums[i]) > 0)
        }
    }
};

File: 26-Remove_duplicates_from_sorted_array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        k=0
        lastChar = '' # Last seen character to determine dupes
        for i in range(len(nums)):
            if(nums[i] != lastChar):
                lastChar = nums[i]
                nums[k] = nums[i]
                k+=1
        nums = nums[:k]
        return k
